### ISSUE TITLE
Allow documentation of dunder methods

### ISSUE THREAD

**Issue by oliversheridanmethven** [jsh9/pydoclint#263]
For a class I may have a dunder method (e.g. `__contains__` or `__exit__`) where I may have some non-trivial behaviour that needs documenting. The tool currently does not check this as far as I can tell.

**Comment by jsh9** [comment]
I think pydoclint already can check this.

I used pydoclint 0.7.5 on this code example:

```python
class MyClass(object):
    def __contains__(self, item: int) -> bool:
        """
        Contains something

        Args:
            item (int2): The item

        Returns:
            bool: the return value
        """
        pass

```

And I got the following violation, as expected:

```
['DOC105: Method `MyClass.__contains__`: Argument names match, but type hints '
 'in these args do not match: item']
```
